app/src/main/java/org/simple/clinic/storage/migrations/RoomMigrationsModule.kt

```diff
@@ -113,7 +113,8 @@ class RoomMigrationsModule {
       migration105: Migration_105,
       migration106: Migration_106,
       migration107: Migration_107,
-      migration108: Migration_108
+      migration108: Migration_108,
+      migration109: Migration_109
```

Comment:
```suggestion
      migration109: Migration_109,
```

Reply:
Kotlin accepts comma in the last element of a list. I suggest adding a comma always so it isn't included in a diff next time an entry is added.

Reply:
For example, if `Migration_108,` had a trailing comma, it wouldn't show up in this diff